code/modules/emotes/definitions/vox.dm

```diff
@@ -1,4 +1,9 @@
 /decl/emote/audible/vox_shriek
-	key ="shriek"
+	key = "shriek"
 	emote_message_3p = "USER SHRIEKS!"
 	emote_sound = 'sound/voice/shriek1.ogg'
+
+/decl/emote/audible/armalis_shriek
+	key ="ashriek"
```

Comment:
Why was this adjusted in one placed but not the other?

Reply:
Good question. Will adjust both.